values.yaml

```diff
@@ -353,6 +353,8 @@ registry:
   # If a secret key is not specified, Helm will generate one.
   # Must be a string of 16 chars.
   secret: ""
+  # This needs to be true when expose.type is not ingress
```

Comment:
I don't think it's related with the expose.type. Could you update the comment according to the document of docker registry https://github.com/docker/distribution/blob/master/docs/configuration.md#http?